practice-areas/accessibility/accessibility-how-to.md

```diff
@@ -1,4 +1,6 @@
-# Accessibility: How To
+---
+title: Accessibility: How To
```

Comment:
@grugnog looks like the title frontmatter doesn't like the colon in the title: https://civicactions-handbook--1123.org.readthedocs.build/en/1123/practice-areas/accessibility/accessibility-how-to/